test/verify/check-static-login

```diff
@@ -640,10 +640,11 @@ account    required     pam_succeed_if.so user ingroup %s""" % m.get_admin_group
         if is_pam_tally2:
             self.sed_file("/common-auth/ { iauth required pam_tally2.so deny=4\n }", "/etc/pam.d/cockpit")
         elif m.image.startswith('debian-') or m.image.startswith('ubuntu'):
-            # enable pam_faillock.  see example in pam_faillock(8), adopted for sss
-            self.sed_file("""s/re.*pam_deny.so/[default=die] pam_faillock.so authfail deny=4/;
-                             s/re.*pam_permit.so/sufficient pam_faillock.so authsucc deny=4/;
-                          """, "/etc/pam.d/common-auth")
+            # enable pam_faillock.  see example in pam_faillock(8)
+            self.sed_file(r"/fallback if no module succeeds/ s/^/"
```

Comment:
`s/^/` reads something like `i`, no?  Alternatively, the other way around, why not just match the line normally?

Not a veto.  I actually think this is fine.

Reply:
Totally agree.